java-frontend/src/main/java/org/sonar/java/se/checks/XxeProcessingCheck.java

```diff
@@ -339,8 +346,13 @@ private ProgramState checkArguments(ProgramState state, Arguments arguments, Xxe
       return state;
     }
 
-    boolean isSettingProperty(ExpressionTree arg0, String propertyName) {
-      return arg0.asConstant(String.class).filter(propertyName::equals).isPresent();
+    boolean isSettingProperty(ProgramState state, ExpressionTree arg0, XxeProperty property) {
+      if (arg0.asConstant(String.class).filter(property.propertyName()::equals).isPresent()) {
```

Comment:
two options here:
* you can replace `property.propertyName()::equals` by `property::isNamed` 
* or you can drop `isNamed(String)`, as I have to feeling that it does the same at the end  (see line below)